internal/policy/verify.go

```diff
@@ -148,7 +152,7 @@ func verifyEntry(ctx context.Context, repo *git.Repository, policy *State, entry
 
 	// No trusted keys => no protection
 	if len(trustedKeys) == 0 {
-		return nil
+		gitNamespaceVerified = true
```

Comment:
nit: Maybe change this comment to `// No trusted keys => allow everything`?

(unless we should fail closed here, in which case we shouldn't be considering this verified)